umpost/um2netcdf.py

```diff
@@ -319,6 +321,66 @@ def fix_latlon_coords(cube, grid_type, dlat, dlon):
     fix_lon_coord_name(longitude_coordinate, grid_type, dlon)
 
 
+def get_default_fill_value(cube):
+    """
+    Get the default fill value for a cube's data's type.
+
+    Parameters
+    ----------
+    cube: Iris cube object.
+
+    Returns
+    -------
+    fill_value: numpy scalar with type matching cube.data
+    """
+    if cube.data.dtype.kind == 'f':
+        fill_value = DEFAULT_FILL_VAL_FLOAT
+
+    else:
+        fill_value = default_fillvals[
```

Comment:
Minor thing to improve the orig code, the top level `from netCDF import ...` can be changed to 'import netCDF'. Then `default_fillvals` can be accessed via the module name to indicate what module it belongs to.

That follows the "best" example here:
https://docs.python-guide.org/writing/structure/#modules

Reply:
Good idea, I've added this in 656d699